plugins/fmcomms1.c

```diff
@@ -1770,20 +1771,20 @@ static GtkWidget * fmcomms1_init(struct osc_plugin *plugin, GtkWidget *notebook,
 	if (ini_fn)
 		load_profile(NULL, ini_fn);
 
-	dds1_freq = dac_data_manager_get_widget(dac_tx_manager, TX1_T1_I, WIDGET_FREQUENCY);
-	dds2_freq = dac_data_manager_get_widget(dac_tx_manager, TX1_T2_I, WIDGET_FREQUENCY);
-	dds3_freq = dac_data_manager_get_widget(dac_tx_manager, TX1_T1_Q, WIDGET_FREQUENCY);
-	dds4_freq = dac_data_manager_get_widget(dac_tx_manager, TX1_T2_Q, WIDGET_FREQUENCY);
+	dds1_freq = dac_data_manager_get_widget(dac_tx_manager, dac_data_manager_dds_tone(0, TONE_1, TONE_I), WIDGET_FREQUENCY);
```

Comment:
these also look like they could parametrized into a for loop;
i could be wrong

Reply:
Yes, they could be. But in a different PR.